app/src/test/java/org/glucosio/android/tools/GlucoseRangesTest.java

```diff
@@ -0,0 +1,191 @@
+package org.glucosio.android.tools;
+
+
+import android.content.Context;
+import android.support.v4.content.ContextCompat;
+
+import org.glucosio.android.BuildConfig;
+import org.glucosio.android.R;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.RuntimeEnvironment;
+import org.robolectric.annotation.Config;
+
+import static junit.framework.Assert.assertEquals;
+import static junit.framework.Assert.assertFalse;
+import static junit.framework.Assert.assertTrue;
+
+@RunWith(RobolectricTestRunner.class)
+@Config(constants = BuildConfig.class, packageName = "org.glucosio.android", sdk = 19)
+public class GlucoseRangesTest {
+
+    private final Context contextMock = RuntimeEnvironment.application.getApplicationContext();
+    private TestGlucoseRanges glucoseRanges = new TestGlucoseRanges(contextMock);
+    private int reading = 65;
+
+    @Test
+    public void expectIsInUnitTestsReturnTrue() {
+        assertTrue(glucoseRanges.isInUnitTests());
+    }
+
+    @Test
+    public void expectIsInUnitTestsReturnFalseByDefault() {
+        assertFalse(new GlucoseRanges().isInUnitTests());
```

Comment:
You don't trust yourself that this is a test glucose ranges class?

Reply:
let's say I'm checking all test scenario possibilities.

Reply:
:) Testing test code is not something that people tend to write